docs/src/content/docs/stack/databases.md

```diff
@@ -0,0 +1,222 @@
+
+[Hono exposes bindings](https://hono.dev/docs/getting-started/cloudflare-workers#bindings) via the [`env` property on the request `Context` object](https://hono.dev/docs/api/context#env). To keep `Context` typing in sync with your bindings, you can pass the `Bindings` types as a generic to Hono app instances or middleware.
+
+In the interest of clarity, HONC templates do so manually, but this can be fragile. A more type-safe approach is to run `wrangler types`, which generates a global `Env` type based on your `.dev.vars` file (and `wrangler.toml`). If you choose this approach though, remember to re-run the command each time you add or modify secrets!
+
+> Cloudflare’s `Env` type shouldn’t be confused with Hono’s `Env`, which is the first generic argument that Hono instances and middleware accept.
+> 
+
+Here is a simple example of Cloudflare secrets and Hono types in action. When properly typed, the `DATABASE_URL` variable can be passed directly to the Neon driver.
+
+```tsx
+import { neon } from "@neondatabase/serverless";
+import { drizzle } from "drizzle-orm/neon-http";
+import * as schema from "./db/schema";
+
+type AppEnv = {
+  Bindings: { DATABASE_URL: string }
+  // The Env type is available globally after running `wrangler types`
+  // Bindings: Env;
+}
+
+const app = new Hono<AppEnv>()
+	.get("/api/users", async (c) => {
+	  const driver = neon(c.env.DATABASE_URL);
+	  const db = drizzle(driver);
+	  
+	  const users = await db.select().from(schema.users);
+	  return c.json({ users });
+	});
+```
+
+In practice though, it’s recommended to initialize the Drizzle client in a dedicated middleware. This makes it easier to keep configurations in sync, and simplifies typing by leveraging Hono’s type inference in handlers.
+
+That’s it! Your database is now live. The next step is to deploy your Worker so you can start making calls to your remote database.
+
+## Supabase
+
+Supabase is a [comprehensive Postgres development platform](https://supabase.com/database). In addition to a managed database, it also provides storage, a user management system, edge functions, and a real-time client. This makes Supabase a great alternative to Firestore, but also means that it has a pricing and development model that may be sub-optimal if you just need a database.
+
+Two notable limitations are that Supabase doesn’t offer free branching, and transactions are only supported via [Database Functions](https://supabase.com/docs/guides/database/functions). Going outside the Cloudflare ecosystem may also introduce some additional latency, especially since it makes colocating databases with Workers more difficult.
```

Comment:
same comment as on neon - could be worthwhile to mention [Hyperdrive](https://developers.cloudflare.com/hyperdrive/) from cloudflare, which is supposed to resolve this issue